src/data/links.json

```
@@ -27,6 +27,10 @@
         {
           "name": "Django Girls Workshop",
           "path": "https://djangogirls.org/en/prague/"
+        },
+        {
+          "name": "Speaker's Dinner",
```

```suggestion
          "name": "Speakers' Dinner",
```
It is the dinner of the speakers (plural) . So I would guess the apostrophe would go after the "s" ?